build/ivysettings.xml

```diff
@@ -15,14 +15,20 @@ SPDX-License-Identifier: Apache-2.0
          Keep this file synchronized with
          https://gitlab.com/sosy-lab/software/java-project-template
     -->
-    <settings defaultResolver="Sosy-Lab"/>
+    <settings defaultResolver="Repos"/>
     <property name="repo.dir" value="${basedir}/repository"/>
     <resolvers>
         <!-- Resolver for downloading dependencies -->
-        <url name="Sosy-Lab" descriptor="required">
-            <ivy pattern="${ivy.repo.url}/[organisation]/[module]/ivy-[revision].xml" />
-            <artifact pattern="${ivy.repo.url}/[organisation]/[module]/[artifact]-[revision](-[classifier]).[ext]" />
-        </url>
+        <chain name="Repos">
+            <url name="Sosy-Lab">
+                <ivy pattern="${ivy.repo.url}/[organisation]/[module]/ivy-[revision].xml"/>
+                <artifact pattern="${ivy.repo.url}/[organisation]/[module]/[artifact]-[revision](-[classifier]).[ext]"/>
+            </url>
+            <url name="Maven-Repo" m2compatible="true">
+                <ivy pattern="https://repo1.maven.org/maven2/[organisation]/[module]/ivy-[revision].xml"/>
+                <artifact pattern="https://repo1.maven.org/maven2/[organisation]/[module]/[revision]/[artifact]-[revision].[ext]"/>
+            </url>
+        </chain>
```

Comment:
Do we need this change or is this just for development?

Reply:
No, it was for development only. Once everything is in our repository it can be removed.

Reply:
I've restored the original version now.